hs_modelinstance/templates/pages/modelinstanceresource.html

```diff
@@ -1,45 +1,69 @@
-{# This template is used as the landing page for TimeSeriesResource #}
+{# This template is used as the landing page for Generic Model Instance resource type #}
 
 {% extends "pages/genericresource.html" %}
-{#{% load geoanalytics_tags pages_tags mezzanine_tags crispy_forms_tags %}#}
+{% load geoanalytics_tags pages_tags mezzanine_tags crispy_forms_tags %}
 {#{% load geoanalytics_tags pages_tags mezzanine_tags comment_tags keyword_tags hydroshare_tags crispy_forms_tags inplace_edit %}#}
 
 {% block extended_metadata %}
     {% if extended_metadata_exists %}
         <div class="row">
-            <div class="col-md-12">
-                <div>
-                    {% if model_output %}
-                        <h4><strong>Model Output:</strong></h4>
-                        <hr style="margin-top: 0px;margin-bottom: 2px">
-                        <div class="row">
-                            <div class="col-sm-4"><strong>Includes output files?</strong></div>
-                            {% if model_output.includes_output == True %}
-                                <div class="col-sm-8">Yes</div>
+        <div class="col-md-12">
+            <div>
+                {% if model_output %}
+                    <h4><strong>Model Output:</strong></h4>
+                    <hr style="margin-top: 0px;margin-bottom: 2px">
+                    <div class="row">
+                        <div class="col-sm-4"><strong>Includes output files?</strong></div>
+                        {% if model_output.includes_output == True %}
+                            <div class="col-sm-8">Yes</div>
+                        {% else %}
+                            <div class="col-sm-8">No</div>
+                        {% endif %}
+
+                    </div>
+                    <hr style="border:0">
```

Comment:
Styles belong in the CSS, not inlined in either the HTML markup or set via Javascript. Please move them before merging this pull request.